test/e2e/storage/testsuites/snapshottable.go

```diff
@@ -282,6 +297,58 @@ func (s *snapshottableTestSuite) DefineTests(driver storageframework.TestDriver,
 					framework.ExpectError(err)
 				}
 			})
+
+			ginkgo.It("start pod, check the claim, check pv, create snapshot, create new pvc from snapshot, new pod to use the pvc (with initial pod attached), validate data on volume", func() {
```

Comment:
A more clear description may be something like "while original volume is still in use"

Reply:
makes sense. Updated it.